<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Yahtzee</class>
 <widget class="QWidget" name="Yahtzee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yahtzee</string>
  </property>
  <widget class="QPushButton" name="rollButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>558</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MyPushButton" name="die1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/dice1.png</normaloff>resources/dice1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyPushButton" name="die0">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/dice1.png</normaloff>resources/dice1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyPushButton" name="die2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/dice1.png</normaloff>resources/dice1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyPushButton" name="die3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/dice1.png</normaloff>resources/dice1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyPushButton" name="die4">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/dice1.png</normaloff>resources/dice1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>451</width>
     <height>661</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="rowCount">
    <number>22</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>x1</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>x2</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>x3</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string notr="true">Ones</string>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>Twos</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>Threes</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>Fours</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>Fives</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="5" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="5" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="5" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>Sixes</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="6" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="6" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="6" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>Top SubTotal</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="7" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="7" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="7" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>Bonus</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="8" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="8" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="8" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="9" column="0">
    <property name="text">
     <string>Top Total</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="9" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="9" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="9" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="10" column="0">
    <property name="text">
     <string>LOWER SECTION</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="10" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="10" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="10" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="11" column="0">
    <property name="text">
     <string>Three of a kind</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="11" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="11" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="11" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="12" column="0">
    <property name="text">
     <string>Four of a kind</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="12" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="12" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="12" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="13" column="0">
    <property name="text">
     <string>Full House</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="13" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="13" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="13" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="14" column="0">
    <property name="text">
     <string>Small Straight</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="14" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="14" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="14" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="15" column="0">
    <property name="text">
     <string>Large Straight</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="15" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="15" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="15" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="16" column="0">
    <property name="text">
     <string>Yahtzee</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="16" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="16" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="16" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="17" column="0">
    <property name="text">
     <string>Chance</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="17" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="17" column="2">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="17" column="3">
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="18" column="0">
    <property name="text">
     <string>Yahtzee Bonus</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="18" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="18" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="18" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="19" column="0">
    <property name="text">
     <string>Lower Total</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="19" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="19" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="19" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="20" column="0">
    <property name="text">
     <string>Grand Total</string>
    </property>
   </item>
   <item row="20" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="20" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="20" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="21" column="0">
    <property name="text">
     <string>Multipliers</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="21" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="21" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="21" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>830</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>Final Score</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
